Documentation/Volumes.md

```diff
@@ -0,0 +1,42 @@
+# Attaching volume
+Capstan supports attaching any number of pre-prepared volumes to the instance. Currenly, you need to prepare
+a volume manually i.e. with external tools, and then export it into a file on your host. During the `capstan run`
+you then provide path to this file together with some metadata.
+
+## Creating a new volume
+You can create a new volume using qemu-img tool:
+
+```bash
+$ qemu-img create -f qcow2 ./volume.img 1G
+```
+
+## Attaching volumes
+Tell Capstan where your volume is and it will attach it to the instance on run:
+
+```bash
+$ capstan run demo --volume ./volume.img
+```
+
+The volume will get attached as `/dev/vblk1` into the unikernel. The `--volume` argument can be repeated
+to attach multiple volumes:
+
+```bash
+$ capstan run demo --volume ./volume1.img --volume ./volume2.img --volume ./volume3.img
+```
+
+Volumes will get attached as `/dev/vblk1`, `/dev/vblk2`, `/dev/vblk3` in the same order as provided (left-to-right).
+
+### Volume metadata
+You can provide volume metadata for Capstan to be able to attach it as desired:
```

Comment:
Despite the fact that you have a reference to the QEMU docs below, I do not understand if the _metadata_ you mention here are directly passed to the QEMU command. If it is, please write this explicitly here that QEMU format for block devices is used (otherwise, it seems like the table below lists the only possible options).

Reply:
Looking at the actual code below, I think it's not a general QEMU format. I will leave the comment here just to be sure.